cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+One can also model the van Keken problem with the volume-of-fluid (VOF) interface tracking 
+algorithm in ASPECT.
```

Comment:
Most readers will probably not know about the VoF method. It might be useful to add a few references here where one could look up what that term means.